tools/js-sdk-release-tools/test/changelogGenerator.test.ts

```diff
@@ -0,0 +1,16 @@
+import { expect, test } from 'vitest'
+import { extractExportAndGenerateChangelog } from '../src/changelog/extractMetaData'
+import path from 'path';
+
+test('HLC -> Modular: Rename', async () => {
+  const oldViewPath = path.join(__dirname, "testCases/operationGroups.old.api.md");
+  const newViewPath = path.join(__dirname, "testCases/operationGroups.new.api.md");
+  const changelog = await extractExportAndGenerateChangelog(oldViewPath, newViewPath);
+  
+  expect(changelog.addedOperationGroup.length).toBe(0);
+  expect(changelog.removedOperationGroup.length).toBe(0);
+  expect(changelog.interfaceParamDelete.length).toBe(1);
+  expect(changelog.interfaceParamDelete[0]).toBe('Interface DataProductsCatalogsOperations no longer has parameter listByResourceXXXGroup');
```

Comment:
i notice the code has the change for operation, do you know what is the difference for operation and parameter here?

Reply:
I'm not quite sure what you mean here. Could you provide more details? I didn't change the operations actually.

Reply:
Here is the comparation for operation: https://github.com/Azure/azure-sdk-tools/blob/7f1d4024c6de375b098f8074da055f0d3380555b/tools/js-sdk-release-tools/src/changelog/changelogGenerator.ts#L1040.

And in your change we have parameter change for listByResourceXXXGroup which is actually an operation.

Reply:
updated